src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+[output]  ~ resource "newrelic_alert_policy" "DemoPolicy" {
+[output]     id         = "1216533"
+[output]    ~ name        = "My Demo Policy" -> "dev: My Demo Policy"
+[output]     # (2 unchanged attributes hidden)
+[output]   }
+[output]
+[output]  # newrelic_alert_policy_channel.ChannelSubs must be replaced
+[output] -/+ resource "newrelic_alert_policy_channel" "ChannelSubs" {
+[output]    ~ channel_ids = [
+[output]      - 4737437,
+[output]     ] -> (known after apply) # forces replacement
+[output]    ~ id     = "1216533:4737437" -> (known after apply)
+[output]     # (1 unchanged attribute hidden)
+[output]   }
+[output]
+[output] Plan: 2 to add, 1 to change, 2 to destroy.
+```
+</Step>
+
+<!-- STORING STATE REMOTELY -->
+
+<Step>
+
+## Storing the State Remotely
+
+At the moment, the state file is local. The next steps will update your Terraform configurations to remove the local state file and use a file stored remotely in Amazon S3.
+
+<Callout variant="tip">
+Since Terragrunt allows you to configure multiple environments, various state files will need to be store in separate S3 buckets, not to override each other.
+</Callout>
+
+Terragrunt allows you to generate files on the fly. Adding the remote_state block into the root level __terragrunt.hcl__ file, you will generate a new file named __backend.tf__.
+```hcl
+ remote_state {
+  backend = "s3"
+  generate = {
+    path      = "backend.tf"
+    if_exists = "overwrite_terragrunt"
+  }
+  config = {
+    bucket = "my-terraform-state"
+
+    key = "${path_relative_to_include()}/terraform.tfstate"
+    region         = "us-east-1"
+    encrypt        = true
+  }
+}
+```
+
+After adding the remote_state block, you will need to give the configuration for the S3 bucket that will store the Terraform state.
+
+<Callout variant="tip">
+To give access to the S3 bucket in your Amazon account, create an IAM user. Give the IAM user access to the S3 bucket storing the terraform state.
+</Callout>
+
+Now, update the remote_state block to provide your S3 bucket and update the key to use the local enviroment variable created previously.
+```hcl
+ remote_state {
+  backend = "s3"
+  generate = {
+    path      = "backend.tf"
+    if_exists = "overwrite_terragrunt"
+  }
+  config = {
+    bucket = "YOUR_S3_BUCKET_NAME"
+
+    key = "envs/${local.env_name}/terraform.tfstate"
+    region         = "us-east-1"
+    encrypt        = true
+    profile		= "YOUR_PROFILE_NAME"
+  }
+}
+```
+
+Next, run `terragrunt plan`.
+
+If you look in the Amazon S3 bucket, you will see a folder named __envs.__ Within the envs folder is a directory called __devs__ containing __terraform.tfstate__ file.
+
+<Callout variant="tip">
+To see the files created by Terragrunt, look inside of the ./enviroments/dev directory. There is a hidden directory named __terragrunt-cache.__ Within the __terragrunt-cachev__ directory, you will see the __backend.tf__ file that Terragrunt generated.
+</Callout>
```

Comment:
```suggestion
<Callout variant="tip">

To see the files created by Terragrunt, look inside of the ./enviroments/dev directory. There is a hidden directory named __terragrunt-cache.__ Within the __terragrunt-cachev__ directory, you will see the __backend.tf__ file that Terragrunt generated.

</Callout>
```

It looks like the embedded markdown isn't parsed correctly and this text is instead rendered verbatim. The space surrounding the content should fix this.

<img width="1010" alt="Screen Shot 2021-02-02 at 4 30 39 PM" src="https://user-images.githubusercontent.com/565661/106680774-033eef80-6574-11eb-9a92-d8e3cfcdaaf4.png">